src/components/include/utils/threads/thread.h

```diff
@@ -90,7 +98,12 @@ void sleep(uint32_t ms);
 Thread* CreateThread(const char* name, ThreadDelegate* delegate);
 void DeleteThread(Thread* thread);
 
+#ifdef QT_PORT
+class Thread : public QObject {
```

Comment:
Necessary investigate in  [SDLWIN-308](https://adc.luxoft.com/jira/browse/SDLWIN-308)